test/mpi-cpp-fortran/base/mpi-cpp-fortran-config.cmake

```diff
@@ -0,0 +1,2 @@
+include("${CMAKE_CURRENT_LIST_DIR}/mpi-cpp-fortran-targets.cmake")
+include("${CMAKE_CURRENT_LIST_DIR}/../BLTSetupTargets.cmake")
```

Comment:
Should this be `include("${CMAKE_CURRENT_LIST_DIR}/BLTSetupTargets.cmake")`

Reply:
Good question.  In the case that the upstream libraries installs the TPL config files, ordering won't matter because TPL targets will be created downstream, and the only targets referenced by the TPL files are their corresponding targets.  In the case the upstream library exports TPL targets, BLTSetupTargets won't even exist inside the install directory, so CMake will emit an error immediately.